packages/ui/src/components/Card/CardFlat/types.ts

```diff
@@ -0,0 +1,6 @@
+import type { Color } from "../../Button";
+
+export type CardColor = Extract<
+  Color,
+  "yellow" | "red" | "blue" | "black" | "purple"
```

Comment:
Shouldn't these be "primary", "secondary", etc?

Reply:
Makes sense to me! Fixed in [update flat cards to use generic colour names](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6143/commits/768b6729d2dbabf67a2387e4608d17cf3671101c)